ts/features/idpay/initiative/details/components/InitiativeCardComponent.tsx

```diff
@@ -23,133 +23,160 @@ type Props = {
-  },
   cardContainer: {
-    height: 230,
-    width: widthPercentageToDP(100)
-  },
-  card: {
-    position: "absolute",
-    alignSelf: "center",
-    width: widthPercentageToDP(100),
-    height: 230
+    backgroundColor: IOColors["blue-50"],
+    borderBottomEndRadius: 24,
+    borderBottomStartRadius: 24,
+    padding: 32,
+    paddingTop: 0,
+    flex: 1
   },
-  imageFull: {
-    zIndex: -1
+  badge: {
+    backgroundColor: IOColors.blue,
+    borderRadius: 24,
+    paddingVertical: 2,
+    paddingHorizontal: 8
   },
-  row: {
-    flexDirection: "row",
-    alignItems: "center",
-    textAlign: "center"
+  bonusLogoContainer: {
+    backgroundColor: IOColors.white,
+    height: 56,
+    width: 56,
+    borderRadius: 8
   },
-  spaced: {
-    justifyContent: "space-between"
+  topCardSection: {
+    flex: 2,
+    alignItems: "center"
   },
-  bottomInitiativeIcon: {
+  bottomCardSection: {
     flex: 1,
-    justifyContent: "flex-end"
-  },
-  paddedMainContent: {
-    zIndex: 1,
-    padding: 32,
-    width: "100%",
-    height: "100%",
-    position: "absolute"
-  },
-  amount: {
-    lineHeight: 32,
-    fontSize: 24,
-    color: IOColors.white,
-    ...makeFontStyleObject("Bold", undefined, "TitilliumWeb")
+    flexDirection: "row",
+    justifyContent: "center"
   },
-  logo: {
-    resizeMode: "contain",
-    height: 65,
-    width: 65
+  bonusStatusContainer: {
+    flexDirection: "row",
+    alignItems: "center"
   },
   consumedOpacity: {
     opacity: 0.5
+  },
+  remainingPercentageSliderContainer: {
+    height: 4,
+    backgroundColor: IOColors.white,
+    width: 100,
+    borderRadius: 4
+  },
+  alignCenter: {
+    alignItems: "center"
   }
 });
 
+type PercentageSliderProps = {
+  percentage: number;
+  isGreyedOut: boolean;
+};
+const BonusPercentageSlider = ({
+  percentage,
+  isGreyedOut
+}: PercentageSliderProps) => {
+  const width = useSharedValue(100);
+  React.useEffect(() => {
+    // eslint-disable-next-line functional/immutable-data
+    width.value = percentage;
+  });
+  const scalingWidth = useAnimatedStyle(() => ({
+    width: withTiming(width.value, { duration: 1000 })
+  }));
+  return (
+    <View style={styles.remainingPercentageSliderContainer}>
+      <Animated.View
+        style={[
+          {
+            width: `${width.value}%`,
+            backgroundColor: isGreyedOut ? IOColors.blue : IOColors["grey-450"],
+            flex: 1,
+            borderRadius: 4
+          },
+          scalingWidth
+        ]}
+      />
+    </View>
+  );
+};
+const formatNumberRightSign = (amount: number) =>
+  `${formatNumberAmount(amount, false)} €`;
 const InitiativeCardComponent = (props: Props) => {
   const { initiativeName, endDate, status, amount, accrued, refunded } =
     props.initiative;
 
   const isInitiativeConfigured = status === InitiativeStatusEnum.REFUNDABLE;
   const toBeRepaidAmount = (accrued || 0) - (refunded || 0);
+  const remainingAmount = (amount || 0) - (accrued || 0);
```

Comment:
<img width="158" alt="Screenshot 2023-03-21 alle 10 39 55" src="https://user-images.githubusercontent.com/6160324/226568053-36053c2c-98d1-4496-90af-63460f17852d.png">

We still have problems here. From BE we have:
- `amount`: the amount still available in the initiative
- `accrued`: total amount accrued with transactions
- `refunded`: amount refunded by wire transfer

Example: initiative with 100 euros availability.

At start we have:
- `amount`: 100
- `accrued`: 0
- `refunded`: 0
- `toBeRefunded`: 0

After 30 euros of transactions (assuming 100% eligibility)
- `amount`: 70
- `accrued`: 30
- `refunded`: 0
- `toBeRefunded`: 30

After first refund:
- `amount`: 70
- `accrued`: 30
- `refunded`: 30
- `toBeRefunded`: 0

In other words, the amount to display is `amount`, but the total (the one used to set the maximum size of the progress bar) should be `amount + accrued`. 

```suggestion
  const totalAmount = (amount || 0) + (accrued || 0);
  const remainingAmount = (totalAmount || 0) - (accrued || 0);
```

Reply:
```suggestion
  const remainingAmount = (amount || 0);
```
As per my previous comment, this should be the correct `remainingAmount`